mne_bids/write.py

```diff
@@ -1210,6 +1212,8 @@ def write_raw_bids(raw, bids_path, events_data=None,
     # BrainVision is multifile, copy over all of them and fix pointers
     elif ext == '.vhdr':
         copyfile_brainvision(raw_fname, bids_path, anonymize=anonymize)
+    elif ext in ['.edf', '.bdf']:
+        copyfile_edf(raw_fname, bids_path, anonymize=anonymize)
```

Comment:
I think I would add a warning here ... a bit like this:

```suggestion
    elif ext in ['.edf', '.bdf']:#
        if (anonymize is not None) and verbose:
            # if daysback results in a year prior to 1985:
            if ...
                warn("Due to EDF/EDF+/BDF file limitations, the requested date
                cannot be written to the meas_date region of the file.
                Setting the meas_date to year 85.
                The recording region will be set to the requested date.")
        copyfile_edf(raw_fname, bids_path, anonymize=anonymize)
```

Reply:
see https://github.com/mne-tools/mne-bids/pull/636#pullrequestreview-534823155 ... we could set to "yy" instead of 85 ... which is the same as saying "look at the recording date, instead of startdate"

Reply:
Added the warning as recommended, though I made it independent of the 'daysback' number for a) simplicity, and b) because `_check_anonymize` already throws a warning if the daysback doesn't put the date earlier than 1925, so people should always be using a daysback that puts them before 1985. Is that okay with you?

Reply:
Yes, I think that's okay! However, I think the warning text can be refined - I'll comment there directly.